README.md

```diff
@@ -9,13 +9,24 @@
 
 It is strongly recommended to use `filestorage.create` to get new `FileStorage` instance because it will create root directory if it doesn't yet exists in file system.
 
-Example:
+Example with existing directory:
 ```js
-const { FileStorage, create } = require('filestorage');
-const storage = new FileStorage({ dir: './root-directory', minCompressSize: 2048 });
-const storage = create({ dir: './root' });
+const fs = require('fs');
+const { FileStorage } = require('filestorage');
+
+const rootDirectory = 'root-directory';
+if (!fs.existsSync(rootDirectory)) fs.mkdirSync(rootDirectory);
```

Comment:
The example says `Example with existing directory:` but you call `mkdirSync`that's not right.